tests/test_quarto_operator.py

```diff
@@ -28,20 +28,20 @@ def quarto(self):
         }
 
     def test_that_knada_operator_image_is_used(self, dag, quarto):
-        container = quarto_operator(dag, "name", "repo", quarto)
+        container = quarto_operator(dag, "name", quarto, "repo")
         assert container.image == "operator-image"
 
     def test_that_personal_operator_image_is_used(self, dag, quarto):
-        container = quarto_operator(dag, "name", "repo", quarto, image="personal-image")
+        container = quarto_operator(dag, "name", quarto, "repo", image="personal-image")
         assert container.image == "personal-image"
 
     def test_that_cmds_are_correct(self, dag, quarto):
-        container = quarto_operator(dag, "name", "repo", quarto)
+        container = quarto_operator(dag, "name", quarto, "repo")
         correct_cmds = ["quarto render quarto.qmd --to html --execute --output index.html -M self-contained:True && " \
                         f"""curl -X PUT -F index.html=@index.html https://datamarkedsplassen.intern.dev.nav.no/quarto/update/{quarto['id']} -H "Authorization:Bearer {quarto['token']}" """]
         assert container.arguments == correct_cmds
 
     def test_that_log_output_is_added_to_cmds(self, dag, quarto):
         with pytest.raises(KeyError) as err:
-            quarto_operator(dag, "name", "repo", {})
+            quarto_operator(dag, "name", {}, "repo")
```

Comment:
```suggestion
            quarto_operator(dag, "name", quarto, "repo")
```
Skulle vel egentlig ikke vært tom dict her? Burde vel egentlig feilet.

Reply:
og den feiler den. Testen forventer `KeyError`